tests/js/spec/views/eventsV2/table/columnEditModal.spec.js

```diff
@@ -408,6 +408,33 @@ describe('EventsV2 -> ColumnEditModal', function () {
         newWrapper.find('RowContainer button[aria-label="Drag to reorder"]')
       ).toHaveLength(0);
     });
+    it('handles equations being deleted', function () {
```

Comment:
And a test for the following?
1. the error message has changed
2. the order has changed